Exercise_C/index.css

```diff
@@ -2,3 +2,30 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+.blue {
+    background-color: blue;
+}
+
+.red {
+    background-color: red;
+    margin-top: 90px;
```

Comment:
You can achieve this exercise without margin-top.
You could define two flex box containers (one that contains blue and green while the other contains red)
These containers would be on top of each other in your HTML code. 

Well done for implementing the answer in your own way.